civihr_default_theme/scss/base/_task-assignments-modal-temp.scss

```diff
@@ -8,15 +8,20 @@ $lighter-grey: #ECEDEE;
 $white: #FFFFFF;
 
 // Opened Modal
-.modal-open {
+[data-ta-documents],
+[data-ta-documents-manager] {
   .modal {
     overflow-y: scroll !important;
 
     .modal-dialog {
-      width: 500px;
+      max-width: 500px;
```

Comment:
This can be achived by using thee small modal of bootstrap,
http://getbootstrap.com/javascript/#modals-sizes